Update wunderground_data.py

uroh opened the pull request:
add Foggy and Fog Early

shtrom commented on custom_components/wundergroundpws/wunderground_data.py:
```diff
@@ -62,6 +62,8 @@ class WUndergroundData:
         'fog': [
             "Fog",
+            "Foggy",
+            "Fog Early",
```
`Fog Early` is not necessary here, as the `Early` gets stripped by using the `condition_modifiers` below, and it ends up looking like `Fog`, which is already coverred.

cytech commented:
noted for next release.
would be nice if weather company API documented the responses so we aren't chasing these down forever...

uroh commented:
ok, i updated to the latest version and removed "Fog early" and added "AM Snw Shwrs" in "snowy". I'm not sure of the classification, however I think it means a snowfall in the morning. Cytech, thanks for the new wunderground code.

uroh commented:
none,
"Unsupported condition string "AM Snw Shwrs". Please update WUndergroundData.condition_map and/or WUndergroundData.condition_modifiers."

uroh commented:
That's right, I removed the "AM". Thank you
